collectd-no-process-metrics.conf

```diff
@@ -0,0 +1,82 @@
+# This file is a variant of the default collectd.conf with no process metrics.
```

Comment:
Actually, I'd suggest `with process metrics disabled`, to imply that the default behavior is to collect them.

Reply:
Updated.